docs/integrations/sources/circleci.md

```diff
@@ -15,11 +15,9 @@ This directory contains the manifest-only connector for [`source-circleci`](http
 | `api_key` | `string` | API Key.  |  |
 | `org_id` | `string` | Organization ID. The org ID found in `https://app.circleci.com/settings/organization/circleci/xxxxx/overview` |  |
 | `start_date` | `string` | Start date.  |  |
-| `project_id` | `string` | Project ID found in the project settings.  |  |
-| `job_id` | `string` | Job ID for fetching information found in URL.  |  |
-| `workflow_id` | `string` | Workflow ID of a project pipeline.  |  |
-| `workflow_name` | `string` | Workflow name for fetching information.  |  |
-| `job_number` | `string` | Job Number of the workflow.  | `2` |
+| `project_id` | `string` | Project ID found in the project settings, Visit `https://app.circleci.com/settings/project/circleci/ORG_SLUG/YYYYY`  |  |
+| `workflow_id` | `string` | Workflow ID of a project pipeline, Could be seen in the URL of pipeline build, Example `https://app.circleci.com/pipelines/circleci/55555xxxxxx/7yyyyyyyyxxxxx/2/workflows/WORKFLOW_ID`  |  |
```

Comment:
Still curious, can we always grab all workflows instead of just one?

Reply:
🤔Oh.. That could also work, I could make parameterized array config for multiple workflow_ids

Reply:
So imagine you have a repo with 10 workflows and you want to sync up data and make a dashboard in BigQuery to visualize how much time, on average, it takes to run your workflows. 

In that scenario, you want to know data about runs of ALL your workflows, not just one. That's my point. 

The connector should grab all defined workflows into workflows stream, and then for each of them, grab all their runs.

Reply:
Yep, Made the config as an array and it could take an array of inputs.